components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsMessageTable/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@wso2-dashboards/widget": "1.2.2",
+    "@wso2-dashboards/widget": "1.4.0",
```

Comment:
With this version bump, package-lock.json should be updated too. Why is it not in this PR? Did you used `npm install @wso2-dashboards/widget@1.4.0` command to bump the version?

Reply:
Fixed at b233d5335fc34cc9a03e24a4944cd39d0f6c3677